nvflare/job_config/base_app_config.py

```diff
@@ -58,9 +58,6 @@ def add_ext_script(self, ext_script: str):
         if not (os.path.isabs(ext_script) or os.path.exists(ext_script)):
             raise RuntimeError(f"Could not locate external script: {ext_script}")
 
-        if not ext_script.endswith(".py"):
-            raise RuntimeError(f"External script: {ext_script} must be a '.py' file.")
-
         self.ext_scripts.append(ext_script)
 
     def add_ext_dir(self, ext_dir: str):
```

Comment:
what's special about "ext_scripts" or "ext_dir", should they just be "resources"?

Reply:
I think they are treated differently when copied into custom.